addons/website_event/views/event_templates_page_registration.xml

```diff
@@ -451,25 +352,137 @@
     </div>
 </template>
 
-<!-- This template is called whenever the "Fold Ticket Details" option is checked on the website (Customize > Fold Tickets Details) -->
-<template id="fold_register_details" inherit_id="website_event.registration_template" active="False" name="Fold ticket details">
-    <xpath expr="//div[@id='o_wevent_tickets']" position="attributes">
-        <attribute name="data-folded-by-default">1</attribute>
-    </xpath>
-    <xpath expr="//div[@id='o_wevent_tickets_collapse']" position="attributes">
-        <attribute name="class">collapse</attribute>
-    </xpath>
-    <xpath expr="//span[hasclass('o_wevent_registration_title')]" position="attributes">
-        <attribute name="class" add="d-none" separator=" "/>
-    </xpath>
-    <xpath expr="//div[hasclass('o_wevent_price_range')]" position="attributes">
-        <attribute name="class">o_wevent_price_range</attribute>
-    </xpath>
-    <xpath expr="//a[hasclass('o_wevent_registration_btn')]" position="attributes">
-        <attribute
-            name="class">btn btn-primary o_wevent_registration_btn collapsed ms-auto</attribute>
-        <attribute name="data-bs-toggle">collapse</attribute>
-    </xpath>
+<template id="modal_ticket_registration" name="Modal for tickets registration">
+    <!-- Modal -->
+    <div class="modal fade" id="modal_ticket_registration" data-bs-backdrop="static" data-bs-keyboard="false" tabindex="-1" aria-labelledby="staticBackdropLabel" aria-hidden="true">
+        <div class="modal-dialog">
+            <div class="modal-content">
+            <div class="modal-header">
+                <div class="o_wevent_registration_title modal-title fs-5" id="staticBackdropLabel">Tickets</div>
+                <div class="o_wevent_price_range ms-2"/>
```

Comment:
You issue with the min/max comes from here. See before this was located inside this block `<t t-if="len(event.event_ticket_ids) &gt; 1">`

So in the override, we knew the block we were replacing was only there if we indeed had tickets. But not anymore.

Even if it's a modal, it's still going to get rendered before being displayed. I would say we can simply keep the behavior here. If there is not more than one ticket, don't show the range as it is not relevant. Using the same t-if condition as below for this div.
```suggestion
                <div t-if="len(event.event_ticket_ids) &gt; 1" class="o_wevent_price_range ms-2"/>
```